controller/search.go

```diff
@@ -83,6 +83,12 @@ func (c *SearchController) Show(ctx *app.ShowSearchContext) error {
 
 	}
 	return application.Transactional(c.db, func(appl application.Application) error {
+		if ctx.Q == nil || *ctx.Q == "" {
+			jerrors, _ := jsonapi.ErrorToJSONAPIErrors(
```

Comment:
We're not logging request/response bodies, but it might have been interesting to get the 'msg' as part of the normal 'complete' log line ?